main.py

```diff
@@ -44,7 +44,8 @@ def main(window, config, algorithm_selection):
     pg.init()
     window = pg.display.set_mode((config['window']['width'], config['window']['height']))
     menu = pg_menu.Menu('Select the triangulation algorithm', config['window']['width'], config['window']['height'], theme=pg_menu.themes.THEME_BLUE)
-    selection = menu.add.selector('Triangulation algorithm :', [(AlgorithmEnum.FLIPPING.value, AlgorithmEnum.FLIPPING), (AlgorithmEnum.INCREMENTAL.value, AlgorithmEnum.INCREMENTAL)])
+    selection = menu.add.selector('Triangulation algorithm :', [(AlgorithmEnum.FLIPPING.value, AlgorithmEnum.FLIPPING), (AlgorithmEnum.INCREMENTAL.value, AlgorithmEnum.INCREMENTAL),
+                                                                (AlgorithmEnum.COVEX_HULL.value, AlgorithmEnum.COVEX_HULL)])
```

Comment:
V tento okamzik bych uz asi pouzil list comprehension:
```suggestion
    selection = menu.add.selector('Triangulation algorithm :', [(algorithm.value, algorithm) for algorithm in AlgorithmEnum])
```